path_config.go

```diff
@@ -63,6 +69,7 @@ type azureConfig struct {
 	TenantID     string `json:"tenant_id"`
 	Resource     string `json:"resource"`
 	Environment  string `json:"environment"`
+	CloudConfig  string `json:"cloud_config`
```

Comment:
The struct tag looks invalid. 

Suggesting:
```suggestion
	CloudConfig  string `json:"cloud_config"`
```